src/util.h

```diff
@@ -22,23 +22,18 @@ typedef struct {
 
 static procOutput check_command(const std::vector<std::string> &args) {
     auto command = subprocess::util::join(args);
-    subprocess::Popen proc(command, subprocess::bufsize{100*1024*1024}, subprocess::output(subprocess::PIPE), subprocess::error(subprocess::PIPE));
+    subprocess::Popen proc(command, subprocess::bufsize{-1}, subprocess::output(subprocess::PIPE), subprocess::error(subprocess::PIPE));
```

Comment:
`-1` is not an obvious value, please document its meaning in a comment.

Reply:
documented